contracts/examples/PayExample.sol

```diff
@@ -0,0 +1,35 @@
+import {Ownable} from '@openzeppelin/contracts/access/Ownable.sol';
+
+pragma solidity 0.8.20;
+
+contract PayExample is Ownable {
+    /**
+     * @dev mapping of keccak256(abi.encode(issuerId, schemaHash) => value
+     */
+    mapping (bytes32 => uint256) private ValueToPay;
+
+    event Payment(uint256 indexed issuerId, string paymentId, uint256 schemaHash);
+
+    constructor() Ownable(_msgSender()) { }
+
+    function setPaymentValue(uint256 issuerId, uint256 schemaHash, uint256 value) public onlyOwner {
+        ValueToPay[keccak256(abi.encode(issuerId, schemaHash))] = value;
+    }
+
+    function pay(string calldata paymentId, uint256 issuerId, uint256 schemaHash) public payable {
+        uint256 requiredValue = ValueToPay[keccak256(abi.encode(issuerId, schemaHash))];
```

Comment:
As discussed in a call, I think we should require here only 1 payment for the same (paymentId, issuerId)

Reply:
added